app/src/main/java/com/babylon/wallet/android/data/gateway/model/TokensAndLsusPricesResponse.kt

```diff
@@ -22,7 +22,7 @@ data class TokensAndLsusPricesResponse(
         @SerialName(value = "resource_address")
         val resourceAddress: String,
         @SerialName(value = "usd_price")
-        val usdPrice: Double,
+        val usdPrice: String,
```

Comment:
my only concern is that the [endpoint](https://token-price-service.radixdlt.com/docs#/default/) doesn't return a String, and I am not sure if that can cause any issues? 
I assume no, but just to keep it in mind.

Reply:
`kotlinx.serialization.json` takes care of it as part of the `Converter.Factory` specified when building `Retrofit` instance. The types such as `Double`, `Int` and other primitive types get converted to String with no issues.

Reply:
you are right! 👌🏽